vfs/fs_utils.cc

```diff
@@ -537,6 +537,33 @@ dirName(const std::string & filename)
 }
 
 /****************************************************************************/
+/* UX FUNCTIONS                                                             */
+/****************************************************************************/
+
+void
+checkWritability(const std::string & url, const std::string & parameterName)
+{
+    // try to create output folder and write open a writer to make sure 
+    // we have permissions before
+    try {
+        Datacratic::makeUriDirectory(url);
```

Comment:
Check latest commit. If nothing throws (ie empty file was created), we remove the empty file before returning.